net-misc/seafile-client/seafile-client-6.2.11.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils gnome2-utils
+
+DESCRIPTION="Seafile desktop client"
+HOMEPAGE="https://github.com/haiwen/seafile-client/ http://www.seafile.com/"
+SRC_URI="https://github.com/haiwen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="shibboleth test"
+
+RDEPEND="net-libs/libsearpc
+	~net-misc/seafile-${PV}
+	dev-libs/libevent
+	dev-libs/jansson
+	dev-libs/openssl:0
+	dev-db/sqlite:3
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtdbus:5
+	shibboleth? ( || ( dev-qt/qtwebengine:5[widgets] dev-qt/qtwebkit:5 ) )"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5
+	test? ( dev-qt/qttest:5 )"
+
+src_prepare() {
+	eapply "${FILESDIR}/${PN}-select-qt5.patch"
+	cmake-utils_src_prepare
+	if use shibboleth ; then
+		if ! has_version "dev-qt/qtwebengine:5[widgets]" ; then
+			sed -i -e 's/IF(WIN32 OR DETECTED_QT_VERSION VERSION_LESS 5.6.0)/IF(TRUE)/' CMakeLists.txt || die
+		fi
+	fi
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_SHIBBOLETH_SUPPORT="$(usex shibboleth)"
+		-DBUILD_TESTING="$(usex test)"
+	)
+	cmake-utils_src_configure
+}
+
+pkg_postinst() {
+	gnome2_icon_cache_update
+}
+
+pkg_postrm() {
+	gnome2_icon_cache_update
```

Comment:
preferably use `xdg_icon_cache_update` from [xdg-utils](https://devmanual.gentoo.org/eclass-reference/xdg-utils.eclass/) instead of deprecate function from [gnome2-utils](https://devmanual.gentoo.org/eclass-reference/xdg-utils.eclass/)

Reply:
Done.